index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <!--TOOLBAR SECTION-->
+        <nav class="toolbar">
+            <div class="my-logo primary-color hover-underline">My Logo</div>
+            <div class=" collapsed_menu primary-color">&#9776;</div>
```

Comment:
Kindly wrap this element in a `<nav>` tag instead of a `<div>`🙏. This is for semantic reasons as the hamburger button links to the mobile menu and is hence a navigation element